clang/test/SemaSYCL/sycl-device-reqd-sub-group-size-template.cpp

```diff
@@ -5,7 +5,9 @@
 template <int SIZE>
 class KernelFunctor {
 public:
```

Comment:
It feels weird to me that existing tests all throw warnings now. I am not sure what is usually done when spellings are changed though. Personally I would just prefer updating these tests to use the new spelling (except where this functionality is actually being checked). @erichkeane thoughts?

Reply:
I could go either way.  There aren't that many tests to just 'fix' the spelling of here, so I'd probably suggest just fixing all the spellings instead.

If this was more widespread (that is, the tests were a pain to change), or a situation where there was a difference in the implementation, I'd consider just using the -Wno flag in the tests.  In this case, I think just changing the spelling in the tests is better than adding the warning/note to ALL of them.

Reply:
That said, probably leave at least 1 to validate the deprecated warning :)

Reply:
Yea. That's what I meant by 'except where this functionality is actually being checked' :) 

@smanna12 sorry but can you make this change? I think it's just cleaner this way.

Reply:
> Yea. That's what I meant by 'except where this functionality is actually being checked' :)
> 
> @smanna12 sorry but can you make this change? I think it's just cleaner this way.

Sure. I will do this.

Reply:
I think you still need to leave ONE example (perhaps add a new example) to validate the new warning/note.

Reply:
yes, right. By mistake, I changed all spellings.

Reply:
Sorry for the multiple submission.